src/components/navigation/navigation.html

```diff
@@ -0,0 +1,120 @@
+    <aside>
+        <h3>Components</h3>
+        <a href="/src/components/avatar/avatar.html">Avatar</a>
+        <a href="/src/components/alert/alert.html">Alert</a>
+        <a href="/src/components/badge/badge.html">Badge</a>
+        <a href="/src/components/button/button.html">Button</a>
+        <a href="/src/components/image/image.html">Images</a>
+        <a href="/src/components/cards/cards.html">Card</a>
+        <a href="#">Input</a>
+        <a href="#">Form</a>
+        <a class="active-tab" href="#">Navigation</a>
+    </aside>
+
+
+
+    <main>
+
+        <!-- ------------ component : avatar ---------------- -->
+
+        <div class=" main-container padding-md">
+            <div class="headline">
+                <h1>Navigation</h1>
+                <div class="description">
+                    Navigation bar contains the menu items through which you can navigate to different pages and
+                    sections of your web application. </div>
+            </div>
+
+            <section class="background-white">
+                <h3>Example of Simple Navigation
+                </h3>
+                <div class="section-description ">
+                    We have added a basic navigation for small works which is used in building this component library
+                </div>
+                <div class="container-row ">
+                    <navigation>
+                        <div class="header-container">
+                            <div class="heading-header">
+                                <h1> Cali-Aesthetics</h1>
+                            </div>
+                            <div class="header-links">
+                                <a class="link-primary">Home</a>
+                                <a class="link-primary">Docs</a>
+                            </div>
+                        </div>
+                    </navigation>
+                </div>
+            </section>
+            <section class="background-offwhite">
+                <h3>Example of Nav Bar for E-Com Websites</h3>
+                <div class="section-description">
+                    You can use this nav bar for making your ecom website.
+                </div>
+
+                <div class="container-row">
+
+                    <navigation class="nav">
+                        <div class="nav-left logo">
+                            <h2 class="nav-logo text-size-lg">Cali Shop</h2>
+                        </div>
+
+                        <div class="nav-center search-bar">
+                            <input type="search" class="input input-search" placeholder="Search" />
+                        </div>
+
+                        <div class="nav-right">
+                            <button class="btn btn-primary login-button text-md">Login</button>
+
+                            <a class="badge-container">
+                                <i class="fas fa-shopping-cart fa-4x">
+                                </i>
```

Comment:
Icons are appearing a little large, you can try with a smaller size.

Reply:
hmm..u r right